### ISSUE TITLE
CDs gone from bartender

### ISSUE THREAD

**Issue by Damaster101** [tullamods/OmniCC#265]
When i updated to 8.0.6 the CDs for atleast Bartender was removed

**Comment by Tuller** [comment]
They appear to be working fine for me. What are your settings?

**Comment by Damaster101** [comment]
Sorry for the dealay, to be honest i don't know my settings.... have never changed them because it has always worked out of the box, up until yeasterday

**Comment by cerogithub** [comment]
I've experienced a similar issue. Specifically, I use the blizzard cooldown numbers for Bartender and OmniCC in other areas. I have found when I sign out (consistent) or change zones / specs (varies) the blizzard option in the config-cache.wtf for SET countdownForCooldowns "1" disappears and I have to manually re-enable "Show Numbers for Cooldowns" every time.

What "fixed" this behavior for me was disabling omni cc.

Turning the option on in the blizzard UI, exiting the game, setting the config-cache.wtf file to read only, then starting the game back up, then re-enable omni cc continues to cause it to fail on bartender. Other cooldowns work fine when this is broken.

I have a rule that disables BT4Button from using OmniCC since i use the default Blizzard cooldown counters.

The only way I could get it to work is to create another group with a rule for BT4Button and reproduce the blizzard cooldown text.
